Secure connection problem

Two secure connection ping test failed, pingWithSecureConnection and pingWithSecureConnectionAndVerification, both throw out an expection about unsupported SSL message.

Related test code:
```@Test
    void pingWithSecureConnection() throws Exception {
        // FIXME
        withNewConnection(connection -> {
                    withStatement(connection, stmt -> {
                        ResultSet resultSet = stmt.executeQuery("SELECT 1");
                        assertTrue(resultSet.next());
                    });
                }, "ssl", "true",
                "ssl_mode", "disabled");
    }

    @Test
    void pingWithSecureConnectionAndVerification() throws Exception {
        // FIXME
        KeyStoreConfig keyStoreConfig = new LocalKeyStoreConfig();

        withNewConnection(connection -> {
                    withStatement(connection, stmt -> {
                        ResultSet resultSet = stmt.executeQuery("SELECT 1");
                        assertTrue(resultSet.next());
                    });
                }, "ssl", "true",
                "ssl_mode", "verify_ca",
                "key_store_type", keyStoreConfig.getKeyStoreType(),
                "key_store_path", keyStoreConfig.getKeyStorePath(),
                "key_store_password", keyStoreConfig.getKeyStorePassword());
    }

%TESTC  1 v2
%TSTTREE2,com.timeplus.jdbc.TimeplusConnectionITest,true,1,false,1,TimeplusConnectionITest,,[engine:junit-jupiter]/[class:com.timeplus.jdbc.TimeplusConnectionITest]
%TSTTREE3,pingWithSecureConnection(com.timeplus.jdbc.TimeplusConnectionITest),false,1,false,2,pingWithSecureConnection(),,[engine:junit-jupiter]/[class:com.timeplus.jdbc.TimeplusConnectionITest]/[method:pingWithSecureConnection()]
%TESTS  3,pingWithSecureConnection(com.timeplus.jdbc.TimeplusConnectionITest)

%ERROR  3,pingWithSecureConnection(com.timeplus.jdbc.TimeplusConnectionITest)
%TRACES 
java.sql.SQLException: Unsupported or unrecognized SSL message
        at com.timeplus.client.NativeClient.connect(NativeClient.java:86)
        at com.timeplus.client.NativeClient.connect(NativeClient.java:52)
        at com.timeplus.jdbc.TimeplusConnection.createNativeContext(TimeplusConnection.java:323)
        at com.timeplus.jdbc.TimeplusConnection.createClickHouseConnection(TimeplusConnection.java:318)
        at com.timeplus.jdbc.TimeplusDriver.connect(TimeplusDriver.java:58)
        at com.timeplus.jdbc.TimeplusDriver.connect(TimeplusDriver.java:51)
        at com.timeplus.jdbc.TimeplusDriver.connect(TimeplusDriver.java:26)
        at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
        at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
        at com.timeplus.jdbc.AbstractITest.withNewConnection(AbstractITest.java:134)
        at com.timeplus.jdbc.TimeplusConnectionITest.pingWithSecureConnection(TimeplusConnectionITest.java:40)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:688)
        at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
        at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:149)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:140)
        at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:84)
        at org.junit.jupiter.engine.execution.ExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(ExecutableInvoker.java:115)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.lambda$invoke$0(ExecutableInvoker.java:105)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
        at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:104)
        at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:98)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6(TestMethodTestDescriptor.java:210)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:206)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:131)
        at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:65)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:139)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
        at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:143)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$7(NodeTestTask.java:129)
        at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:127)
        at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:126)
        at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:84)
        at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
        at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:108)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:88)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:54)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:67)
        at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:52)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:96)
        at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:84)
        at org.eclipse.jdt.internal.junit5.runner.JUnit5TestReference.run(JUnit5TestReference.java:98)
        at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:40)
        at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:529)
        at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:757)
        at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:452)
        at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:210)
Caused by: javax.net.ssl.SSLException: Unsupported or unrecognized SSL message
        at java.base/sun.security.ssl.SSLSocketInputRecord.handleUnknownRecord(SSLSocketInputRecord.java:457)
        at java.base/sun.security.ssl.SSLSocketInputRecord.decode(SSLSocketInputRecord.java:175)
        at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:111)
        at java.base/sun.security.ssl.SSLSocketImpl.decode(SSLSocketImpl.java:1511)
        at java.base/sun.security.ssl.SSLSocketImpl.readHandshakeRecord(SSLSocketImpl.java:1421)
        at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:456)
        at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:427)
        at com.timeplus.client.NativeClient.connect(NativeClient.java:77)
        ... 77 more`

--- a/timeplus-native-jdbc/src/test/java/com/timeplus/jdbc/TimeplusConnectionITest.java
+++ b/timeplus-native-jdbc/src/test/java/com/timeplus/jdbc/TimeplusConnectionITest.java
@@ -14,8 +14,8 @@
 
 package com.timeplus.jdbc;
 
-// import com.timeplus.jdbc.tool.LocalKeyStoreConfig;
-// import com.timeplus.settings.KeyStoreConfig;
+import com.timeplus.jdbc.tool.LocalKeyStoreConfig;
+import com.timeplus.settings.KeyStoreConfig;
 import org.junit.jupiter.api.Test;
 
 import java.sql.ResultSet;
@@ -36,31 +36,29 @@ public class TimeplusConnectionITest extends AbstractITest {
 
     @Test
     void pingWithSecureConnection() throws Exception {
-        // FIXME
-        // withNewConnection(connection -> {
-        //             withStatement(connection, stmt -> {
-        //                 ResultSet resultSet = stmt.executeQuery("SELECT 1");
-        //                 assertTrue(resultSet.next());
-        //             });
-        //         }, "ssl", "true",
-        //         "ssl_mode", "disabled");
+        withNewConnection(connection -> {
+                    withStatement(connection, stmt -> {
+                        ResultSet resultSet = stmt.executeQuery("SELECT 1");
+                        assertTrue(resultSet.next());
+                    });
+                }, "ssl", "true",
+                "ssl_mode", "disabled");
     }
 
     @Test
     void pingWithSecureConnectionAndVerification() throws Exception {
-        // FIXME
-        //KeyStoreConfig keyStoreConfig = new LocalKeyStoreConfig();
+        KeyStoreConfig keyStoreConfig = new LocalKeyStoreConfig();
 
-        // withNewConnection(connection -> {
-        //             withStatement(connection, stmt -> {
-        //                 ResultSet resultSet = stmt.executeQuery("SELECT 1");
-        //                 assertTrue(resultSet.next());
-        //             });
-        //         }, "ssl", "true",
-        //         "ssl_mode", "verify_ca",
-        //         "key_store_type", keyStoreConfig.getKeyStoreType(),
-        //         "key_store_path", keyStoreConfig.getKeyStorePath(),
-        //         "key_store_password", keyStoreConfig.getKeyStorePassword());
+        withNewConnection(connection -> {
+                    withStatement(connection, stmt -> {
+                        ResultSet resultSet = stmt.executeQuery("SELECT 1");
+                        assertTrue(resultSet.next());
+                    });
+                }, "ssl", "true",
+                "ssl_mode", "verify_ca",
+                "key_store_type", keyStoreConfig.getKeyStoreType(),
+                "key_store_path", keyStoreConfig.getKeyStorePath(),
+                "key_store_password", keyStoreConfig.getKeyStorePassword());
     }
 
     @Test

